lib/igniter/project/config.ex

```diff
@@ -294,6 +276,31 @@ defmodule Igniter.Project.Config do
     end)
   end
 
+  def remove_configuration(igniter, file_name, app_name) do
```

Comment:
We may want to give this a name that indicates that it effectively can only remove an "entire application's" configuration from a configuration, like `remove_app_configuration` or something like that. Additionally, the same application could be configured multiple times in this file, so this will need to recursively check again, only returning `:error` if it never found a match.